docs/operations/metrics.md

```diff
@@ -350,7 +350,7 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 
 ### JVM
 
-These metrics are only available if the `JVMMonitor` module is included.
+These metrics are only available if the `JvmMonitor` module is included.
```

Comment:
Nit:
```suggestion
These metrics are only available if the `JvmMonitor` module is included in `druid.monitoring.monitors`.
```
You could even link to this https://druid.apache.org/docs/latest/configuration/index.html#enabling-metrics